packages/account-v2/src/utils/apiErrorCodes.ts

```diff
@@ -0,0 +1,3 @@
+export const API_ERROR_CODES = {
+    INVALID_OTP: 'InvalidOTP',
```

Comment:
**NIT**: In accounts-v2 we are trying to enforce Airbnb style guide as much as possible, so to declare enums we are not using SNAKE upper case for object keys  
```
export const POI_SUBMISSION_STATUS = {
    complete: 'complete',
    selecting: 'selecting',
    submitting: 'submitting',
} as const;
```